client/components/happychat/notices.jsx

```diff
@@ -77,7 +79,7 @@ class Notices extends Component {
 
 const mapState = state => ( {
 	isServerReachable: isHappychatServerReachable( state ),
-	chatStatus: getHappychatStatus( state ),
+	chatStatus: getHappychatChatStatus( state ),
```

Comment:
Just a nitpick but if we are going to keep only one selector around I think `getHappychatStatus` might make more sense.

Reply:
I might agree as well, but I still don't know for sure :) Selector naming is confusing now, and I'd like to come up with a naming schema that makes sense for the whole Happychat. I plan to do that in future PRs after all of them are in one place, so it's easier to compare them.

Reply:
I like what @nosolosw did in leaving `getHappychatChatStatus` — there's also a "connection" status (`connected`, `disconnected`, etc), so just saying `getHappychatStatus` is ambigious. This is much more explicit that we're talking about "chat" statuses, e.g. `assigned`, `abandoned`, etc